### ISSUE TITLE
Refresh not working for linked references

### ISSUE THREAD

**Issue by Meh-S-Eze** [stevencohn/OneMore#480]
**Describe the bug**
When using the refresh button for linked references Receiving Security Notice and then "You'll need a new app to open this onemore link. 

**To Reproduce**
1. Add linked references
2. Hit refresh
3. 

**Expected behavior**
Expect that the page will refresh instead nothing happens

**Screenshots**
![image](https://user-images.githubusercontent.com/66741217/158700944-227957e7-dcb1-4064-81da-4213e405e91f.png)
![image](https://user-images.githubusercontent.com/66741217/158700954-489adfb9-e48e-48ab-9aa9-5f1273955581.png)

**Environment (if applicable):**
 - OneNote Version:Microsoft® OneNote® for Microsoft 365 MSO (16.0.14326.20702) 32-bit 
 - OneMore Version 4.16

**Additional context**

[OneMore.log](https://github.com/stevencohn/OneMore/files/8274834/OneMore.log)

**Comment by stevencohn** [comment]
Did you install OneMore as an administrator?

If you know how to, you could download and run iq.ps1 script. This is an installation qualification script. It doesn't change anything; it only reports the OneMore related entries in the Windows System Registry. This will tell you whether there are invalid or missing keys that should have been created by the installer.

**Comment by stevencohn** [comment]
This is an example of running iq.ps1

<img width="784" alt="image" src="https://user-images.githubusercontent.com/9964757/158721376-da7442a0-bf16-425c-a8fe-d69aa5f09932.png">

**Comment by Meh-S-Eze** [comment]
Had to do some learning before I could follow this.  I tried to copy the raw code from the script and save it using atom with the appropriate extension.  The file it self shows as being associated with power shell.  In powershell I tried to run the script but recieved the below. 

![image](https://user-images.githubusercontent.com/66741217/171723671-5c91ec81-0fd0-441e-ae04-a2d069318957.png)

**Comment by stevencohn** [comment]
This is odd especially considering it's complaining about line 252 and there are only 247 lines in the file! I'm guessing maybe you have an old version of PowerShell. 

Regardless, this is likely related to all of the other installer issues that people have reported so I'm hoping will be resolved by that work.

**Comment by julie777** [comment]
I just had the same dialog pop up today. In my case I had moved the page that the favorite linked to a new notebook.
I had to recreate the link.
Is there a refresh for favorites?
I know that OneNote has two types of links and that one of them will keep itself up-to-date. Maybe OneMore favorites could actually be stored on a hidden OneNote page so that OneNote would update links. (This is an idea that I have been thinking about for a while, not just for Favorites, but for storing macro settings, etc.)
